recipes/libglvnd/all/conanfile.py

```diff
@@ -0,0 +1,171 @@
+from conans import Meson, ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import functools
+import os
+import textwrap
+
+required_conan_version = ">=1.45.0"
+
+class LibGlvndConan(ConanFile):
+    name = "libglvnd"
+    description = "The GL Vendor-Neutral Dispatch library"
+    license = "custom"
```

Comment:
```suggestion
    license = "LicenseRef-LICENSE"
```

Use `LicenseRef-<license-file>` if there is no SPDX identifier. We should probably document this one

Ref. https://github.com/conan-io/conan-center-index/pull/4928#pullrequestreview-614562909